docs/asyncapi.yaml

```diff
@@ -34,6 +52,49 @@ components:
       payload:
         $ref: '#/components/schemas/ReturnMessage'    
   schemas:
+    DWHCalculation:
+        type: object
+        properties:
+          calculationDate:
+            $ref: '#/components/schemas/CalculationDate'
+          calculationReference:
+            $ref: '#/components/schemas/CalculationReference'
+          invoiceNumber:
+            $ref: '#/components/schemas/InvoiceNumber'
+          frn:
+            $ref: '#/components/schemas/FRN'
+          sbi:
+            $ref: '#/components/schemas/SBI'
+          type:
+            $ref: '#/components/schemas/Type'
+          updated:
```

Comment:
@marctemp It looks like this property is missing an object type, causing parsing to fail. Kindly check. Thanks

Reply:
Well spotted -- fixed (👍